src/allmydata/interfaces.py

```diff
@@ -934,6 +935,13 @@ def get_current_size():
         """
 
 
+class IVerifyNode(IFilesystemNode):
+    """
+    I am a node which represents an opaque repairable object. I can be checked
+    and repaired but neither read nor written.
```

Comment:
Interfaces specify what can at least be done, so this should not say "but neither read nor written".